docker/scripts/provision-core.sh

```
@@ -119,6 +119,17 @@ explorer_sync() {
   fi
 }
 
+chart_sync() {
+  if is_sync_on $SYNC_CHART_SNAPSHOT; then
+    echo "Sync chart snapshot"
+    if [ "$NETWORK" == "mainnet" ]; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, == in place of = is undefined.

(SC3014)